pkg/reconciler/openshift/tektonconfig/extension.go

```diff
@@ -103,6 +131,117 @@ func (oe openshiftExtension) Finalize(ctx context.Context, comp v1alpha1.TektonC
 }
 
 // configOwnerRef returns owner reference pointing to passed instance
-func configOwnerRef(tc v1alpha1.TektonComponent) metav1.OwnerReference {
-	return *metav1.NewControllerRef(tc, tc.GroupVersionKind())
+func configOwnerRef(tc v1alpha1.TektonInstallerSet) metav1.OwnerReference {
+	return *metav1.NewControllerRef(&tc, tc.GetGroupVersionKind())
+}
+
+func createInstallerSet(ctx context.Context, oc versioned.Interface, ta *v1alpha1.TektonConfig,
+	releaseVersion, component, installerSetName string) error {
+
+	is := makeInstallerSet(ta, installerSetName, releaseVersion)
+
+	createdIs, err := oc.OperatorV1alpha1().TektonInstallerSets().
```

Comment:
let us move all `installerSet` related functionality into a separate file within this package (or to  the common package if that makes more sense). 

At present, we are calling methods from rbac.go from this file then later calling functions in this file from rbac.go.